src/Header/Header.test.jsx

```diff
@@ -133,7 +133,7 @@ describe("Header", () => {
 	});
 
 	describe("Skip link target", () => {
-		it("Doesn't render skip link target when it already exists on the page", () => {
+		it("Doesn't create a skip-link target if there's already one in the document", () => {
```

Comment:
Is this test redundant now because it _never_ creates a skip link now?

Reply:
quite right sir